generators/javascript/index.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+const Generator = require("yeoman-generator");
+const chalk = require("chalk");
+const yosay = require("yosay");
+const path = require("path");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.argument("appname", { type: String, required: false });
+    this.option("yarn", {
+      description: "Use Yarn as the package manager"
+    });
+
+    this.useYarn = this.options.yarn;
+    this.name = this.options.appname || "myapp";
+    this.description = "My cool app";
+    this.version = "1.0.0";
+    this.apiRoot = "/api/v1";
+    this.specification = "openapi_3";
+  }
+
+  async prompting() {
+    // Have Yeoman greet the user.
+    this.log(
+      yosay(
+        `Welcome to the supreme ${chalk.red(
+          "generator-fullstack-bootstrapper"
```

Comment:
small thing but should we change this to generator-front-to-back ?

Reply:
Yes! Forgot to change that, I'll change and make a commit